extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -535,6 +538,13 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     return {resolver, rejector, promise};
   }
 
+  /** @return {!Object<string,string|boolean|number>} */
+  getPageParameters_() {
```

Comment:
Not totally crazy about the naming of this, as we already have googlePageParameters and pageLevelParameters. Not sure what would be better names though...

Reply:
Yeah I hear you... googlePageParameters are those shared by AdSense and GPT.  I renamed pageLevelParameters to googPageLevelParameters